wherobots/db/connection.py

```diff
@@ -209,6 +215,13 @@ def __execute_sql(self, sql: str, handler: Callable[[Any], None]) -> str:
             "statement": sql,
         }
 
+        if self.__store:
+            request["store"] = {
+                "format": self.__store.format.value if self.__store.format else None,
+                "single": str(self.__store.single),
+                "generate_presigned_url": str(self.__store.generate_presigned_url),
```

Comment:
Are you sure you need the string cast here? Wouldn't the JSON serialization later on take care of that?